net/netatalk/Makefile

```diff
@@ -8,81 +8,103 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=netatalk
-PKG_VERSION:=3.2.5
+PKG_VERSION:=4.0.3
 PKG_RELEASE:=1
-PKG_LICENSE:=GPL-2.0-or-later
 
-#PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=@SF/netatalk
-PKG_HASH:=57de9a7ed411029d6176e429a14ef314460251d8aebe7139aeadc35633d9584b
+PKG_HASH:=beaf77962fe15deffb5a09853b2f1a07c308f8bf27369398d62b46d9be7ce83f
 
-PKG_BUILD_PARALLEL:=1
-PKG_INSTALL:=1
-PKG_FIXUP:=autoreconf
+# Unicode DB file required for compilation
+UC_DATA_FILE:=UnicodeData.txt
+UC_DATA_URL:=https://www.unicode.org/Public/16.0.0/ucd/
+UC_DATA_HASH:=ff58e5823bd095166564a006e47d111130813dcf8bf234ef79fa51a870edb48f
 
+PKG_MAINTAINER:=Antonio Pastor <antonio.pastor@gmail.com>
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
 PKG_CPE_ID:=cpe:/a:netatalk:netatalk
 
-PKG_BUILD_DEPENDS:=libevent2
-
 include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+include $(INCLUDE_DIR)/meson.mk
+
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=0
+PKG_FIXUP:=autoreconf
 
 define Package/netatalk
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=Filesystem
-  DEPENDS:=+libattr +libdb47 +libgcrypt +libopenssl +libevent2
-  TITLE:=netatalk
+  TITLE:=Netatalk - Open Source file server that implements the Apple Filing Protocol (AFP)
```

Comment:
In my case, this Title is too long and thus it is not disaplayed in make menuconfig.

Reply:
Will revise. I got too creative. It just said "netatalk" before. Can make it half of that no problem.

Reply:
Revised. Tested. Fits OK in menuconfig.